src/test/java/lotto/domain/LottoMachineTest.java

```diff
@@ -11,6 +11,6 @@ public class LottoMachineTest {
     @Test
     @DisplayName("로또 발급 검증")
     void issue() {
-        assertThat(new LottoMachine(lottoNumbers()).issue(new Payment(10000)).size()).isEqualTo(10);
+        assertThat(new LottoMachine(lottoNumbers()).automaticIssue(10).size()).isEqualTo(10);
```

Comment:
```suggestion
        assertThat(new LottoMachine(lottoNumbers()).automaticIssue(10)).hasSize(10);
```

이런 메서드도 적극 활용해 보아요~ (리뷰하며 보니 다른 곳에서는 사용 중이시네요 😮)